Add scenario multi-line with documentation - macOS (#1486)
wazuh/wazuh-qa: docs/tests/integration/test_logcollector/test_macos/index.md

- **[Test macOS format only future events](test_macos_format_only_future_events.md)**: Check if `only-future-events`
  works correctly for macOS log format.
  

> - **[Test macOS multiline values](test_macos_multiline_values.md)**: Check if `wazuh-logcollector` gather corrrectly generated 
> unified logging system multi-line events.

Reviewer: ```suggestion
- **[Test macOS multiline values](test_macos_multiline_values.md)**: Check if `wazuh-logcollector` correctly collects multiline events from the unified logging system.
```

Reviewer: I think that this would be clearer.
